mitodl/mitxonline
# Sort courses on home page by date ascending

#### Pre-Flight checklist



#### What are the relevant tickets?
fixes #365

#### What's this PR do?
Sort courses on the home page by date ascending
- The course with the soonest, future start date should appear at the top of the page. #367
- Courses with no future start date, should be at the bottom of the page

#### How should this be manually tested?
Just add 5 - 10 feature courses (some with course runs in the past and rest with future nearest to the current date course runs) on the home page in cms and see that courses order

#### Screenshots (if appropriate)
**NOW**

<img width="1439" alt="Screenshot 2022-01-12 at 18 04 07" src="https://user-images.githubusercontent.com/4043989/149148496-93d809ca-75c1-4810-af68-af113baf3b3e.png">

<img width="1037" alt="Screenshot 2022-01-14 at 12 49 53" src="https://user-images.githubusercontent.com/4043989/149471689-99062138-f88f-4e59-8823-d045b3a63958.png">

**BEFORE**

<img width="1440" alt="Screenshot 2022-01-12 at 14 19 45" src="https://user-images.githubusercontent.com/4043989/149148416-8a9d88a6-dd62-4870-8d71-4559746f93b7.png">
